csp-guru/datapackage.json

```diff
@@ -157,33 +191,239 @@
+                        "description": "Number of distinct heliostats or dishes",
+                        "type": "number"
+                    },
+                    {
+                        "name": "Mirror_area_per_heliostat_or_dish",
+                        "description": "the aperture area of the heliostat mirrors in square meters",
+                        "type": "number"
+                    },
+                    {
+                        "name": "Fresnel_line_width_m",
+                        "description": "The width of the line of a fresnel type CSP plant",
+                        "type": "number"
+                    },
+                    {
+                        "name": "Fresnel_number_of_lines",
+                        "description": "Technical parameter of fresnel type CSP plants: the number of fesnel lines",
+                        "type": "number"
+                    },
+                    {
+                        "name": "Fresnel_line_length_m",
+                        "description": "Technical parameter of fresnel type CSP plants: the length of the fesnel lines",
+                        "type": "number"
+                    },
+                    {
+                        "name": "Trough_solar_collector_area_m2",
+                        "description": "Technical parameter of trough type CSP plants: the aperture area of the trough solar collector assemblie (SCA)",
+                        "type": "number"
+                    },
+                    {
+                        "name": "Trough_SCA_length_m",
+                        "description": "Technical parameter of trough type CSP plants: the length of the trough solar collector assemblie (SCA)",
+                        "type": "number"
+                    },
+                    {
+                        "name": "Trough_number_of_loops",
+                        "description": "Technical parameter of trough type CSP plants: the number of loops",
+                        "type": "number"
+                    },
+                    {
+                        "name": "Trough_number_of_modules_per_SCA",
+                        "description": "Technical parameter of trough type CSP plants: the number of modules per solar collector assemblie",
+                        "type": "number"
+                    },{
+                        "name": "Trough_number_of_SCA",
+                        "description": "Technical parameter of trough type CSP plants: the number of solar collector assemblies",
+                        "type": "number"
+                    },
+                    {
+                        "name": "Trough_number_of_SCA_per_loop",
+                        "description": "Technical parameter of trough type CSP plants: the number of solar collector assemblies per loop",
+                        "type": "number"
+                    },
+                    {
+                        "name": "Construction+jobs-years",
```

Comment:
Avoid `+` in the variable name if possible. I'm not sure whether it's a problem, but I'd rather not try.